src/main/resources/config/application-dev.yml

```diff
@@ -105,6 +105,18 @@ jhipster:
         # Token is valid 24 hours
         token-validity-in-seconds: 86400
         token-validity-in-seconds-for-remember-me: 2592000
+
+# ===================================================================
+# Sentry specific properties
+# How to configure Sentry Spring Boot: https://docs.sentry.io/clients/java/config/
+# Full reference is available at: https://github.com/getsentry/sentry-java/blob/main/sentry-spring-boot-starter/src/main/java/io/sentry/spring/boot/SentryProperties.java
+# ===================================================================
+sentry:
+  dsn:
+  stacktrace:
+    app:
+      packages: org.mskcc.oncokb.curation;
+
 # ===================================================================
 # Application specific properties
```

Comment:
Under application specific properties, could we add
```
frontend:
  sentryDsn: 
```

Reply:
Do you mind if I do sentry-dsn since it looks like we're using dashes for space for everything else?

Reply:
Yes, please use the dash.